build.rs

```diff
@@ -1,27 +1,42 @@
+#[cfg(feature = "buildtime_bindgen")]
+extern crate bindgen;
 extern crate cmake;
 
 fn main() {
-    let mut cfg = cmake::Config::new("openvr");
+    let mut config = cmake::Config::new("openvr");
 
-    // Work around broken cmake build
-    #[cfg(windows)]
-    cfg.cxxflag("/DWIN32");
+    if cfg!(target_os = "macos") {
```

Comment:
For correctness, uses of `cfg!` in this build script should probably be replaced with e.g. `env::var("CARGO_CFG_TARGET_OS")`.